pootle/apps/pootle_app/management/commands/list_projects.py

```diff
@@ -31,10 +35,9 @@ def handle_noargs(self, **options):
     def list_projects(self, **options):
         """List all projects on the server."""
 
-        revision = options.get("modified_since", 0)
```

Comment:
I know you dislike temporal variables however this looks like a valid use case to me.